task/run-script-oci-ta/0.1/run-script-oci-ta.yaml

```diff
@@ -107,6 +112,7 @@ spec:
       args:
         - use
         - $(params.SOURCE_ARTIFACT)=/var/workdir/source
+        - $(params.PREFETCH_ARTIFACT)=/var/workdir/cachi2
```

Comment:
Should I use `/var/workdir/prefetch` here now? Or wait for the changes on prefetch dependencies?

Reply:
sounds good :+1:

Reply:
I'll note that the `/cachi2` part of `--volume /var/workdir/cachi2:/cachi2` is the more impactful decision here since, as @Zokormazo pointed out in https://github.com/konflux-ci/build-definitions/pull/2205#discussion_r2053395888, users may sometimes depend on the directory name explicitly.

But it makes sense to me to keep this consistent with the buildah task for now and then make the changes all at once